dio/CHANGELOG.md

```diff
@@ -7,7 +7,8 @@
 - Fix `IOHttpClientAdapter.onHttpClientCreate` Repeated calls
 - `IOHttpClientAdapter.onHttpClientCreate` has been deprecated and is scheduled for removal in
   Dio 6.0.0 - Please use the replacement `IOHttpClientAdapter.createHttpClient` instead.
-- Using `CancelToken` no longer closes and re-creates `HttpClient` for each request when `IOHttpClientAdapter` is used. 
+- Using `CancelToken` no longer closes and re-creates `HttpClient` for each request when `IOHttpClientAdapter` is used.
+- Fix timeout handling for browsers on `connectTimeout` and `receiveTimeout`.
```

Comment:
```suggestion
- Fix timeout handling for browser `receiveTimeout`.
```
This PR actually should not have any effect on the connectTimeout which is handled manually.